### DISCUSSION TITLE
Afet Sonrası İşlerini Kaybedenlere İş İmkanı Hakkında

### DISCUSSION THREAD

**Discussion by sonerce** [acikkaynak/deprem-yardim-projesi#57] [Ideas]
Merhabalar,
Afet sonrası işini kaybeden depremzede dostlarımıza iş imkanlarının sunulabileceği ve onların da başvurabileceği, hızlı yanıt alabileceği bir mini platform yapılabilir mi? Twitterda bir çok depremzede olup iş arayan yazılımcı arkadaş var. Malum sitelerde dönüşler hemen olmuyor başvurular olduğu için. Networkü geniş olan beyfendiler, hanfendiler iş arayan depremzedelere networklerindeki açık pozisyonları önerebilir mesela.

deprem.io üzerinde bu proje canlanabilir.

Not: Kötüye kullanımı engellemek için kamu kurumlarından doğrulama alınabilir (nüfus kaydı vb.)

**Comment by WintersCoder** [comment]
Hayatta kalan deprem mağdurlarını şu an ev ve iş bulma kaygısı sardı. 
İş bulmalarını sağlamak için uygulama ile hayatların dram olmasını engelleriz.

On binlerce kişi işsiz kaldı. Farklı şehirlerde hayat kurmak zorunda olanlar olacaktır.
Bu insanların, iş arayan diğer %10 ile aynı koşullarda olmadığı aşikar, evleri de yok.

Hangi illerde yaşamak istiyorlar belirtebilsinler.
Yetkinliklerine göre iş bulmalarını sağlayacak bir portal yapalım.
Mevcut CV lerini veya LinkedIn profillerini ekleyebilirler. Parsing ile yetkinliği biz de çıkarabiliriz.

Uzmanları ile çalışıp hangi illerde hangi yeteneklere ihtiyaç var yönlendirme yapabiliriz.
Profesyonel IK firmalarından destek alabiliriz.
İş kurmaları konusunda da destek veren de olacaktır.

**Reply by Arikitokurin** [reply]
![image](https://user-images.githubusercontent.com/124596763/218257485-95a121dd-1595-4c8d-bf81-4769619dfcf9.png)

**Reply by Arikitokurin** [reply]
![image](https://user-images.githubusercontent.com/124596763/218257612-77abf457-cf83-4883-947d-62815a561178.png)

**Comment by kayisyaparemin** [comment]
selamlar bu platformu gördüm az önce https://www.afet.in zaten yapılmış.

**Reply by sercangoger** [reply]
yararlı linkler bölümüne eklenebilir. çok başarılı

**Reply by WintersCoder** [reply]
Geliştirme takımına ulaşabilir miyiz? Eklenebilecek özellikler çok. Ayrıca işe yerleştirme firmalarından destek alabiliriz. Mağdur lara öncelik verilebilir, veya bu kişilerin işe alımında ücretsiz hizmet verilebilir.

**Reply by coderserdar** [reply]
![Screenshot_2](https://user-images.githubusercontent.com/11051271/218436422-4ac84839-c4ed-423a-849b-8fad6ed4bb24.png)

Şu anda site devre dışı galiba hocam

**Reply by zbetulaltundal** [reply]
Site hala devre dışı, bir sorun var sanırım.

**Reply by zekeriyamulbay** [reply]
Tekrar yayina alinmis https://twitter.com/burakdonertas/status/1625411155739942912

**Comment by DID0101** [comment]
merhabalar ben MERN stack biliyorum size yadimci olabilirim

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Bu talebiniz Legal ekibin kontrolünden geçti.
Notları: "Teknik altyapısı oluşturulduktan sonra alınacak açık rıza ve onay ile mümkün bir çalışma"

**Reply by sonerce** [reply]
Çok teşekkürler. Umarım insanların işine yarabilen nice projeler üretebiliriz.

**Comment by zekeriyamulbay** [comment]
@ramazansancar ve @kilicfurkan selam. Bu proje icin cok basit bir MVP hazirladim.

Videosunu burada bulabilirsiniz: https://www.loom.com/share/3d02ee25fec04ec7a834f9095fee9a1a

Onu acik mi diye gormek icin 1 saat vakit ayirdim. Tasarim ve fonksiyonellik olarak bu gozle bakin lutfen.

Mentorun uzmanligina ve sehrine gore mentee onerisi
<img width="1240" alt="Screenshot 2023-02-13 at 9 23 41 pm" src="https://user-images.githubusercontent.com/67113841/218577864-01bd35fb-8313-4bcf-b9fe-56ac03dd1d81.png">

CV Havuzu
<img width="1243" alt="Screenshot 2023-02-13 at 9 25 25 pm" src="https://user-images.githubusercontent.com/67113841/218578017-edfc048b-93b9-4ed3-ad24-955904df61d3.png">

Kullanici profili goruntuleme
<img width="1328" alt="Screenshot 2023-02-13 at 9 26 41 pm" src="https://user-images.githubusercontent.com/67113841/218578280-a14d4e36-b741-4212-b5ae-a054d0a640a3.png">

Tabi ki form yonetimi
<img width="751" alt="Screenshot 2023-02-13 at 9 28 28 pm" src="https://user-images.githubusercontent.com/67113841/218578587-98673249-d7ec-447b-8e9c-b7f9117d0d2a.png">

**Reply by sonerce** [reply]
Beğendim. Fakat açık kaynaklı olması devamlılığı açısından önemli diye düşünüyorum. Softr'nin yarın bir gün kapanmayacağını vs. bilemeyiz. Ben böyle düşünüyorum. 

Yine de çok anlaşılır ve şık olmuş.

**Comment by zekeriyamulbay** [comment]
Burak afet.in'i tekrar yayina almis arkadaslar oradan devam edilebilir: https://twitter.com/burakdonertas/status/1625411155739942912
